rpc/js/package.json

```diff
@@ -25,6 +32,7 @@
   "scripts": {
     "prepare": "./etc/prepare.sh",
     "typecheck": "tsc --noEmit",
+    "lint": "eslint '.*.cjs' '**/*.{ts,js,cjs}' --quiet",
```

Comment:
I would recommend against using `--quiet` unless we need to; better to start off noisy than not noisy enough